ui/src/app/cohort-search/redux/actions/service.ts

```diff
@@ -193,7 +193,10 @@ export class CohortSearchActions {
     this.requestCounts('groups', groupId, request);
   }
 
-  requestTotalCount(): void {
+  /**
+   * @param outdatedGroup: string
+   */
+  requestTotalCount(outdatedGroupId?: string): void {
```

Comment:
Is this a chance to write a test case for this function?

Reply:
Sure is.  I wrote a few, should get us along the road to testing the redux machinery a bit more.